neurom/apps/morph_stats.py

```diff
@@ -143,6 +154,9 @@ def extract_stats(neurons, config):
 
     {config_path}
     """
+    if isinstance(neurons, (str, Path)) or (isinstance(neurons, list) and
+                                            all(isinstance(nrn, (str, Path)) for nrn in neurons)):
+        neurons = nm.load_neurons(neurons)
```

Comment:
I'm going to see if I can make `load_neurons` lazy instead of doing this.

Reply:
this is such a nice line of code!

Reply:
do you want to move this line to the load_neurons? So that if a collection of neuron is passed, it will not load it?

Reply:
this is a pain, I'll leave it like that